soroban-env-host/src/lib.rs

```diff
@@ -48,8 +48,8 @@ pub use host::testutils::call_with_suppressed_panic_hook;
 #[cfg(any(test, feature = "testutils"))]
 pub use host::ContractFunctionSet;
 pub use host::{
-    metered_map::MeteredOrdMap, metered_vector::MeteredVector, Host, HostError, LedgerInfo, Seed,
-    SEED_BYTES,
+    metered_map::MeteredOrdMap, metered_vector::MeteredVector, ContractInvocationEvent, Host,
```

Comment:
This should be guarded by `#[cfg(any(test, feature = "testutils"))]`

Reply:
[699a4cc](https://github.com/stellar/rs-soroban-env/pull/1229/commits/699a4ccfe276005168b11a00f00c0a58bcca9dd6)